boltdesignsystem/bolt
# DS-548 Listing teaser custom status numbers

## Jira

https://pegadigitalit.atlassian.net/browse/DS-548

## Summary

Adds a `status.number_items` prop to render custom status numbers in Listing Teaser.

## Details

1. Twig and schema added for status number.
2. New prop added to render custom status numbers instead of replies and views.
3. Replaced all instances of `status.replies` and `status.views` with `status.number_items`.
4. Added deprecation text for `status.replies` and `status.views` in schema.

## How to test

Run the branch locally and check the custom status number doc under Listing Teaser. Make sure the new prop works as expected.
